terraform/modules/data-share-service/prometheus.tf

```diff
@@ -11,11 +11,47 @@ resource "random_password" "prometheus_password" {
   special = false
 }
 
-data "aws_iam_policy" "grafana_prometheus_access" {
-  name = "AmazonPrometheusQueryAccess"
+data "aws_iam_policy_document" "grafana_prometheus_access" {
+  statement {
+    actions = [
+      "aps:GetLabels",
+      "aps:GetMetricMetadata",
+      "aps:GetSeries",
+      "aps:QueryMetrics",
+      "aps:ListRules",
+      "aps:ListAlertManagerSilences",
+      "aps:ListAlertManagerAlerts",
+      "aps:GetAlertManagerStatus",
+      "aps:ListAlertManagerAlertGroups",
+      "aps:PutAlertManagerSilences",
+      "aps:DeleteAlertManagerSilences",
+    ]
+    resources = [aws_prometheus_workspace.prometheus.arn]
+    effect    = "Allow"
+  }
+}
+
+resource "aws_iam_policy" "grafana_prometheus_access" {
+  name   = "${var.environment}-grafana-prometheus-access"
+  policy = data.aws_iam_policy_document.grafana_prometheus_access.json
 }
 
 resource "aws_iam_role_policy_attachment" "grafana_prometheus_access" {
   role       = var.grafana_task_role_name
-  policy_arn = data.aws_iam_policy.grafana_prometheus_access.arn
+  policy_arn = aws_iam_policy.grafana_prometheus_access.arn
+}
+
+resource "aws_prometheus_alert_manager_definition" "prometheus" {
+  workspace_id = aws_prometheus_workspace.prometheus.id
+  definition   = <<EOF
+alertmanager_config: |
+  route:
+    receiver: 'default'
+  receivers:
+    - name: 'default'
+      sns_configs:
+      - topic_arn: ${aws_sns_topic.sns_alarm_topic.arn}
```

Comment:
Is this the topic we've subscribed the email address to?

Reply:
Yep :)